src/app/content/highlights/utils/addCurrentPageToSummaryFilters.ts

```diff
@@ -0,0 +1,30 @@
+import { AppServices, MiddlewareAPI,  } from '../../../types';
+import { page as pageSelector } from '../../selectors';
+import { stripIdVersion } from '../../utils/idUtils';
+import { setSummaryFilters } from '../actions';
+import * as select from '../selectors';
+import { getHighlightLocationForPage } from '../utils';
+
+const addCurrentPageToSummaryFilters = ({
+  dispatch, getState,
+}: MiddlewareAPI & AppServices) => {
+  const state = getState();
+  const page = pageSelector(state);
+  const locations = select.highlightLocations(state);
+  const filters = select.summaryFilters(state);
+  if (!locations.size || !page || typeof(window) === 'undefined') {
```

Comment:
after thinking more about the other comments about moving this logic to occur when the modal is opened, not on navigation, we probably want to keep this check as you have it written now. 

the idea is that when someone opens the dialog it should show them their current chapter, not necessarily every chapter they've navigated through since loading the page, the same applies to the chapter of the page that was loaded.

so forget the previous comment.